core/src/test/java/org/mobilitydata/gtfsvalidator/testgtfs/GtfsStopTableContainer.java

```diff
@@ -58,4 +58,9 @@ public Optional<GtfsStop> byPrimaryKey(String id, String subId) {
   public boolean isRequired() {
     return true;
   }
+
+  @Override
+  public boolean isRecommended() {
```

Comment:
I'm not sure if I'm missing something here, but is there any reason why we would need this for a test related to `GtfsStop`? Also, there seems to be `no usages found` according to IntelliJ.

Reply:
I believe I hit some errors where the existing tests were failing without this logic, I will double check to confirm.

Reply:
Confirmed, I get this error when running `gradlew test` without this change. 

```
> Task :core:compileTestJava FAILED
/Users/kclough/projects/jarvus/gtfs-validator/core/src/test/java/org/mobilitydata/gtfsvalidator/testgtfs/GtfsStopTableContainer.java:26: error: GtfsStopTableContainer is not abstract and does not override abstract method isRecommended() in GtfsTableContainer
public class GtfsStopTableContainer extends GtfsTableContainer<GtfsStop> {
       ^
1 error

FAILURE: Build failed with an exception.
```

Reply:
Ah, right, it has to be implemented because the class inherits the abstract method. All good.